reconcile/aus/models.py

```diff
@@ -16,3 +19,26 @@ class ClusterUpgradeSpec(BaseModel):
 class OrganizationUpgradeSpec(BaseModel):
     org: AUSOCMOrganization
     specs: list[ClusterUpgradeSpec]
+
+
+class ConfiguredUpgradePolicyConditions(BaseModel):
```

Comment:
i'm not sure i understand the `Configured` prefix of all dataclasses?
these dataclasses are the representation of the desired state, right? maybe a `Spec` suffix?

Reply:
It's not the desired state, it's the configuration of the Upgrade scheduler. The desired state is derived from the configuration, i.e. the soak days and the current state. Hence I changed it from desired to configuration.

Reply:
IMHO the schema property https://github.com/app-sre/qontract-schemas/blob/main/schemas/openshift/cluster-1.yml#L230 should not be called sth like `upgradeSchedulerSettings`. Not sure if a rename could make sense here. 
Right now it causes issues in the codebase since we have name overlap for different concepts.